<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPU Scheduling</string>
  </property>
  <widget class="QFrame" name="Add">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>21</y>
     <width>311</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="AddButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>72</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>进程名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>优先权</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>运行时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="plus_process">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>添加进程</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="RunTime">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="priority">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>46</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>7</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>内存大小</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="memsize">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="Reserve">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>291</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="PoolQueue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>后备队列</string>
    </property>
   </widget>
   <widget class="QPushButton" name="JobpushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>作业调度</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="PoolplainTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>261</width>
      <height>17</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>进程名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>运行时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>优先级</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>状态</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="SortpushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>排序</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="Ready">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>301</width>
     <height>221</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="ReadyQueue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>就绪队列</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="ReadyplainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>261</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="CPUpushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU调度</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>281</width>
      <height>17</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>进程名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>运行时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>优先权</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>状态</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>指针</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="SortReadypushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>排序</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>591</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Suspendedlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>挂起队列</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="SuspendedplainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SuspendedpushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LosspushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>挂起：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>解挂：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SuspendedlineEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LosslineEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入需要操作的进程的PID</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>281</width>
     <height>221</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="CPUlabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CPU</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>进程名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>运行时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>优先级</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="NumOfTrack">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>46</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>6</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>道数：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PIDlineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="RunTimelineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="PrioritylineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU调度方法：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="CPUSchedulingMethod">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>46</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>时间片：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="TimeSlice">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>46</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>8</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>488</y>
     <width>571</width>
     <height>2</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="MemoryTableView">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>260</y>
     <width>291</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="PartitionTableView">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>50</y>
     <width>291</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
